package.json

```diff
@@ -4,7 +4,10 @@
   "description": "Koa web app framework",
   "main": "lib/application.js",
   "scripts": {
-    "test": "make test"
+    "lint": "eslint lib test",
+    "test": "NODE_ENV=test mocha --require should --require should-http test/application/* test/context/* test/request/* test/response/*",
+    "test-cov": "NODE_ENV=test istanbul cover ./node_modules/.bin/_mocha -- -u exports --require should --require should-http test/application/* test/context/* test/request/* test/response/* --bail",
```

Comment:
`_mocha` is already in the path in scripts, no need for `.node_modules/.bin/_mocha`, this should work fine:
```
"test-cov": "NODE_ENV=test istanbul cover _mocha -- -u exports --require should --require should-http test/application/* test/context/* test/request/* test/response/* --bail",
```